src/neynar-api/neynar-api-client.ts

```diff
@@ -1251,10 +1252,49 @@ export class NeynarAPIClient {
    */
   public async searchUser(
     q: string,
-    viewerFid: number
+    viewerFid: number,
+    options?: { limit?: number, cursor?: string}
```

Comment:
Adding a limit in the example would be nice. So new users would know how to pass optional params.